mqtt/mqtt-bridge/config/default.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
this file is no longer needed. Default values are defined in `mqtt-generic` or `mqtt-edgehub` crates, and they can be different for "generic" or "edgehub" features.